<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatasetView</class>
 <widget class="QDialog" name="DatasetView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3840</width>
    <height>2160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>938</width>
          <height>487</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="toolTip">
              <string>The JSON key for this dataset. Usually it is a simplified version of the Name. E.g. The dataset with Name Giops Daily could have the Key giops_day. No spaces are allowed here.</string>
             </property>
             <property name="text">
              <string>Dataset Key:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="toolTip">
              <string>URL to the aggregated.ncml file for this dataset. This is passed to THREDDS.</string>
             </property>
             <property name="text">
              <string>URL:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_6">
             <property name="toolTip">
              <string>Detailed information about the dataset goes here. This is presented in the Ocean Navigator.</string>
             </property>
             <property name="text">
              <string>Help:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="toolTip">
              <string>&quot;Nicely&quot; formatted name of the dataset. This is presented on the Ocean Navigator webpage.</string>
             </property>
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="toolTip">
              <string>Dataset attribution. Presented in bottom right corner of the map and in all plot figures in the Ocean Navigator.</string>
             </property>
             <property name="text">
              <string>Attribution:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="toolTip">
              <string>URL to the climatology aggregated.ncml file. This is passed to THREDDS.</string>
             </property>
             <property name="text">
              <string>Climatology URL:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_5">
             <property name="toolTip">
              <string>Time scale of the data inside this dataset. Days, hours, monthly.</string>
             </property>
             <property name="text">
              <string>Quantum:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditKey"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditName"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditAttribution"/>
           </item>
           <item row="6" column="1">
            <widget class="QTextEdit" name="textEditHelp"/>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="comboBoxQuantum">
             <property name="maxVisibleItems">
              <number>5</number>
             </property>
             <item>
              <property name="text">
               <string>Hour</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Day</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Month</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditClima"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditURL"/>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Hover over each item label for more info.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DatasetView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DatasetView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
